src/server/pps/server/pipeline_state_driver.go

```diff
@@ -40,7 +40,7 @@ type PipelineStateDriver interface {
 	Watch(ctx context.Context) (<-chan *watch.Event, func(), error)
 	// list all PipelineInfos
 	ListPipelineInfo(ctx context.Context, f func(*pps.PipelineInfo) error) error
-	GetPipelineInfo(ctx context.Context, name string, version int) (*pps.PipelineInfo, error)
+	GetPipelineInfo(ctx context.Context, projectName, pipelineName string, version int) (*pps.PipelineInfo, error)
```

Comment:
should this just take a `Pipeline`?

Reply:
Great catch! The consequent change enabled me to convert `ppsdb.VersionKey` to take a pipeline too, which is awesome.